README.md

```diff
@@ -438,9 +438,9 @@ TypeScript is a typed superset of JavaScript that compiles to plain JavaScript.
 
 #### TypeScript General Resources
 * [TypeScript](http://www.typescriptlang.org/) Official Website for TypeScript
-* [REPL](http://www.typescriptlang.org/Playground) Official TypeScript REPL that runs entirely in your browser
+* [REPL](http://www.typescriptlang.org/Playground///) Official TypeScript REPL that runs entirely in your browser
 * [TypeScript Repository (GitHub)](https://github.com/Microsoft/TypeScript) Official GitHub Repo for TypeScript
-* [TSD](http://definitelytyped.org/tsd/) TypeScript Definition manager for DefinitelyTyped
+* [TSD](http://angular-meteor.com/angular2) TypeScript Definition manager for DefinitelyTyped
```

Comment:
This does not seem right, it redirects to `tsdpm.org` which seems to be obsolete.